data/102/191/573/102191573.geojson

```diff
@@ -1010,6 +1010,7 @@
         85633813
     ],
     "wof:concordances":{
+        "gn:id":"6255146",
```

Comment:
possible to adjust this, from string to int?

Reply:
fixed